src/components/screens/delegates/delegatesListView/delegateRow.test.js

```diff
@@ -20,46 +18,30 @@ describe('DelegateRow', () => {
     },
     voteToggled: () => {},
   };
-  const options = {
-    context: { store },
-    childContextTypes: { store: PropTypes.object.isRequired },
-  };
 
   it('should have a list item with class name of "pendingRow" when props.data.pending is true', () => {
-    const wrapper = mount(
-      <DelegateRow {...props} voteStatus={pendingStatus} />,
-      options,
-    );
+    const wrapper = mount(<DelegateRow {...props} voteStatus={pendingStatus} />);
     const expectedClass = 'pendingRow';
     const className = wrapper.find('ul').prop('className');
     expect(className).to.contain(expectedClass);
   });
 
   it.skip('should have a list item with class name of "votedRow" when voteStatus.unconfirmed and confirmed are true', () => {
```

Comment:
if we are not using this one we can remove it. What do you think?

Reply:
This file will be cleaned up in https://github.com/LiskHQ/lisk-hub/issues/2088